lib/tasks/convert.rake

```diff
@@ -66,16 +66,50 @@ namespace :db do
   namespace :convert do
     desc 'Convert/import production data to development.   DANGER Deletes all data in the development database.   Assumes both schemas are already migrated.'
     task :prod2dev => :environment do
+      # dynflow:migrate migrates the db configured for the current rails env
+      # In this case, we need to make sure it migrates development
+      env_bak = ::Rails.env
+      Rake::Task['dynflow:migrate'].invoke
```

Comment:
shouldn't this just run with `development` env? since we are always migrating `production`=>`development`.

Reply:
Well, we are always migrating `production => development`, regardless of what `::Rails.env` is set to. `dynflow:migrate` is picky about this and always migrates the db associated with the current rails environment.